### DISCUSSION TITLE
installation issues

### DISCUSSION THREAD

**Discussion by agnes-sharan** [google/timesketch#2905] [Q&A, quick solutions, support]
Hi I've been following [install](https://github.com/google/timesketch/blob/master/docs/guides/admin/install.md) for the installation. However, when I am trying to do step 3, I'm confused which docker compose to use. I copied over the docker file from the docker/dev folder in the root but am getting an error that credentials are not found. I also tried to run docker up from the folder in itself. Is there some step I'm missing?

<img width="578" alt="Screenshot 2023-09-23 at 12 53 52 PM" src="https://github.com/google/timesketch/assets/30827029/5646647e-57e2-4088-a6a8-45b22b273096">

**Comment by jkppr** [comment]
Hi @agnes-sharan did you manage to get Timesketch running by now?
I will close #2920 since I do not expect it to be any issue with the Timesketch code base and will try to support you here in getting it up and running if still necessary.

Have you checked the troubleshooting guide already? https://timesketch.org/guides/admin/troubleshooting/

The docker compose command mentioned in step 3 of your linked guide should be run at `/opt/timesketch/` if this is the folder you have installed timesketch to. 

Can you provide me with the following information:
1. What device, OS, architecture are you trying to deploy Timesketch on?
2. Are you trying to deploy an operational or a dev instance?
  * https://timesketch.org/guides/admin/install/ VS https://timesketch.org/developers/getting-started/

**Comment by jkppr** [comment]
Closing due to no reaction.
